functions/concepts_filesystem/index.php

```diff
@@ -0,0 +1,36 @@
+<?php
+/**
+ * Copyright 2020 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// [START functions_concepts_filesystem]
+
+use Psr\Http\Message\ServerRequestInterface;
+
+function listFiles(ServerRequestInterface $request): string
+{
+    $directory = dirname(__FILE__);
+    $contents = scandir($directory);
```

Comment:
nit: circuitous code.

`__DIR__` is equivalent to `dirname(__FILE__)`. Further, `scandir('.')` is equivalent to `scandir(__DIR__)`. Since this is sometimes ambiguous, I suggest we directly use __DIR__.


```suggestion
    $contents = scandir(__DIR__);
```

Reply:
Done - I was wondering if such a constant existed, thanks!